### PR TITLE
Fix oozie.sh - Debian 10 backports repo issue.

### PR DESCRIPTION
**Pull request by kuldeepkk-dev**

### CODE REVIEW

**Comment by kuldeepkk-dev** [comment]
Tests are passing on 2.X and failing on 2.2 which are unrelated to this change.

File: oozie/oozie.sh [hunk 1]
```diff
@@ -679,6 +697,10 @@ EOF
 
 
 function main() {
+  #Remove debian backports
+  if [[ ${OS_NAME} == debian ]]; then
```
Comment (prince-cs) [inline oozie/oozie.sh:701]: Adding a check for dataproc version is also required, as the issue only affected debian 10 which is being used by dataproc 2.0

**Review by prince-cs — CHANGES_REQUESTED** [review]

File: oozie/oozie.sh (hunk 1 above)
Comment (kuldeepkk-dev) [reply oozie/oozie.sh]: Done, added condition.

**Review by prince-cs — APPROVED** [review]
LGTM

**Review by cjac — APPROVED** [review]
LGTM

**Comment by cjac** [comment]
2.1-debian11 fails with

Error from server (NotFound): pods "presubmit-2-1-debian11-32dbb602-ca42-47a1-aff2-cd66385a2c81" not found

This looks like an infrastructure failure in k8s...

**Pull request merged**